frappe/public/js/frappe/form/grid.js

```diff
@@ -790,7 +790,7 @@ export default class Grid {
 	}
 
 	set_value(fieldname, value, doc) {
-		if (this.display_status !== "None" && doc?.name && this.grid_rows_by_docname[doc.name]) {
+		if (this.display_status !== "None" && doc?.name && this?.grid_rows_by_docname[doc.name]) {
```

Comment:
Hey, I believe that `this` cannot be undefined here because of the `this.display_status` just before.

The more likely fix is:

```diff
-   this?.grid_rows_by_docname[doc.name]
+   this.grid_rows_by_docname?.[doc.name]
```

Reply:
Ah yeah meant to put it there, didn't realised, thanks for pointing out!